src/data-display/progress-bar/PProgressBar.vue

```diff
@@ -25,12 +27,34 @@ export default {
         },
         color: {
             type: String,
-            default: 'rgba(theme(\'colors.primary1\')',
+            default: undefined,
+        },
+        height: {
+            type: String,
+            default: '0.875rem',
+        },
+        gradient: {
+            type: Object,
+            default: undefined,
+        },
+        borderRadius: {
+            type: String,
+            default: '0.125rem',
         },
     },
     setup(props: ProgressBarProps) {
+        const linearGradientProperty = `linear-gradient(90deg, ${props.gradient?.startColor} ${props.gradient?.gradientPoint}%, ${props.gradient?.endColor} 100%)`;
```

Comment:
```suggestion
        const linearGradientProperty = computed(() => `linear-gradient(90deg, ${props.gradient?.startColor} ${props.gradient?.gradientPoint}%, ${props.gradient?.endColor} 100%)`);
```

props.gradient 변경사항이 반영되어야 하므로, computed로 감싸야 정상동작 할 것으로 보여요!